.github/workflows/tests.yml

```diff
@@ -0,0 +1,175 @@
+# Workflow to build and test wheels
+name: Run tests
+
+on: push
+
+jobs:
+
+  test_skimage_linux:
+    name: Run tests linux-cp${{ matrix.python-version }}-${{ matrix.OPTIONS_NAME }}
+    runs-on: ubuntu-latest
+    env:
+        BUILD_DOCS: 1
+        TEST_EXAMPLES: 0
+
+    strategy:
+      # Ensure that a wheel builder finishes even if another fails
+      fail-fast: false
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+        PIP_FLAGS: [""]
+        MINIMUM_REQUIREMENTS: [0]
+        QT: ["PyQt5"]
+        WITH_PYSIDE: [0]
+        BUILD_DOCS: [1]
+        PYTHONOPTIMIZE: [0]
+        TEST_EXAMPLES: [0]
+        OPTIONAL_DEPS: [0]
+        OPTIONS_NAME: [""]
```

Comment:
This ends up being a little confusing in the output:

```
Run tests / Run tests linux-cp3.6- (push)
                                  ^
```

I'm not sure where it's used and why the default is blank (perhaps "default" might be better?)

Reply:
@terencehonles thanks for noticing! I don't know about this and would rather not mess with it until after this PR is merged. 😬 As it is painful enough to iterate here...

Reply:
@terencehonles it is used to have a different name for each job. "default" is a good idea, thanks!